engine/redis-helper.js

```diff
@@ -19,17 +28,10 @@ const commands = {
 ['set', 'get', 'del', 'exists', 'sismember', 'hmset', 'hget', 'smembers',
  'sadd', 'hgetall', 'srem', 'select', 'flushdb', 'quit']
 .forEach(name => {
-  commands[name] = function redisFunction() {
-    const args = Array.prototype.slice.call(arguments);
+  commands[name] = function redisFunction(...args) {
```

Comment:
YES - I REMOVED THAT LINT FROM CONFIG - JUST HAVEN'T REVERTED ALL THE CODE - THANKS FOR FINDING IT
Sry for caps